src/runtime/cli/kata-check.go

```diff
@@ -306,8 +306,9 @@ func genericHostIsVMContainerCapable(details vmContainerCapableDetails) error {
 }
 
 var kataCheckCLICommand = cli.Command{
-	Name:  checkCmd,
-	Usage: "tests if system can run " + project,
+	Name:    "check",
+	Aliases: []string{"kata-check"},
```

Comment:
cool! Didn't know that we can have aliases here.

Reply:
I even tried for the options `--kata-config` and `--kata-show-default-config-paths`, but Go wouldn't let me, even though the documentation for `cli.StringFlag`/`cli.BoolFlag` suggests that aliases can be set there too. Getting the options to work can be a follow-up MR